newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/index.js

```diff
@@ -156,13 +172,8 @@ const GetStartedSection = ({
       durationInMinutes: 5,
       locked: false, // First phase is never locked
       // Phase is disabled if complete or if there's a running tutorial
-      disabled:
-        !!currentlyRunningInAppTutorial ||
-        (!!userProgress &&
-          !!userProgress.progress &&
-          userProgress.progress[0] === 100),
-      progress:
-        userProgress && userProgress.progress ? userProgress.progress[0] : 0,
+      disabled: isFullTutorialRunning || isFlingTutorialPartComplete(0),
```

Comment:
Same here, shouldn't we disabled any in app tutorial if there's already one running, whatever its type is

Reply:
I thought that it was cool to be able to restart the tutorial in one click, without having to click on "quit tutorial".
But I think it's best to disable everything yes. I'll update that